install_wsl.ps1

```diff
@@ -0,0 +1,104 @@
+# WSL installation.
+#
+# Copyright (C) 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019 Gregory W. Chicares.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+#
+# http://savannah.nongnu.org/projects/lmi
+# email: <gchicares@sbcglobal.net>
+# snail: Chicares, 186 Belle Woods Drive, Glastonbury CT 06033, USA
+
+# If you have UnauthorizedAccess error, run this command in PowerShell:
+# Set-ExecutionPolicy Unrestricted
+#
+# Return back to the Restricted policy after installing:
+# Set-ExecutionPolicy Restricted
+#
+# More info: https:/go.microsoft.com/fwlink/?LinkID=135170
+
+
+# Enable Microsoft-Windows-Subsystem-Linux feature.
+if (([Security.Principal.WindowsPrincipal] `
+  [Security.Principal.WindowsIdentity]::GetCurrent() `
+  ).IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator) -And
```

Comment:
I'm not really sure what happens here, but it looks like the condition will fail if the user doesn't have administrative rights. But this wouldn't be correct because it'd mean that we'd silently fail in this case and I think we need at least an error message about not having the necessary permissions or something like this.